ArduCopter/Copter.cpp

```diff
@@ -485,11 +485,18 @@ void Copter::update_batt_compass(void)
 }
 
 // Full rate logging of attitude, rate and pid loops
-// should be run at 400hz
-void Copter::fourhundred_hz_logging()
+// should be run at loop rate
+void Copter::loop_rate_logging()
 {
     if (should_log(MASK_LOG_ATTITUDE_FAST) && !copter.flightmode->logs_attitude()) {
         Log_Write_Attitude();
+        Log_Write_PIDS(); // only logs if PIDS bitmask is set
```

Comment:
I guess I don't understand why the split out when the overall outcome is the same but using a little more flash?

Reply:
this allows PIDS to be logged independently of ATT....this functionality is implied in all bitmask bits....see line 512

Reply:
If PIDS are not linked to ATT there is no rate. So if all attitude was turned off we could log 10hz PIDs I guess, but that is fairly pointless, its too slow.

Reply:
Yeah, I would be inclined to just leave them together - there is no PIDS bit

Reply:
sorry but you are mistaken...there is a PIDs log mask bit and its currently being used